jasmine-standalone-3.6.0 (1) 2/src/bowling.js

```diff
@@ -0,0 +1,73 @@
+'use strict';
+
+class Bowling{
+  constructor() {
+    this.current_score = 0;
+    this.bonus1 = -1;
```

Comment:
the -1 - does this mean if it wasn't a bonus ?  Maybe something more clean or a comment else it may become arbitrary for different individuals :).